packages/lesswrong/components/posts/PostsItemTooltipWrapper.jsx

```diff
@@ -22,4 +22,5 @@ const PostsItemTooltipWrapper = ({hover, anchorEl, stopHover, children, post}) =
     </React.Fragment>
 }
 
-registerComponent('PostsItemTooltipWrapper', PostsItemTooltipWrapper, withHover)
+registerComponent('PostsItemTooltipWrapper', PostsItemTooltipWrapper,
+  withHover({pageElementContext: "postItemTooltip"}, ({post}) => ({postId: post._id})))
```

Comment:
```suggestion
  withHover({pageElementContext: "postItemTooltip"}, ({post}) => ({postId: post?._id})))
```

Reply:
Curious about this one. Why would post be null on a postItemTooltip, and if it is, shouldn't it not fail silently?